examples/interdomain/nsm_istio/nse-auto-scale/iptables-map.yaml

```diff
@@ -2,9 +2,6 @@
 - -N NSM_PREROUTE
 - -A NSM_PREROUTE -j ISTIO_REDIRECT
 - -I PREROUTING 1 -p tcp -i {{ .NsmInterfaceName }} -j NSM_PREROUTE
-- -N NSM_OUTPUT
-- -A NSM_OUTPUT -j DNAT --to-destination {{ index .NsmSrcIPs 0 }}
-- -A OUTPUT -p tcp -s 127.0.0.6 -j NSM_OUTPUT
```

Comment:
OK... after some review, I think you are likely correct to remove this.  Do be extra careful to check that traffic coming into the istio pod from the istio-cluster does in fact get correctly proxied to the workload pod in the nsm-cluster.

Reply:
Of course, I already checked it. But I'll double check